README.md

```diff
@@ -1,33 +1,103 @@
+To [read an analog input](https://www.arduino.cc/en/Tutorial/AnalogReadSerial) from a [potentiometer](http://www.resistorguide.com/potentiometer/), the following code and schematic can be used.
 
-[wiring picture]
+![Potentiometer Schematic](https://www.arduino.cc/en/uploads/Tutorial/AnalogReadSerial_BB.png)
 
-Now that the sensor is wired, we've got to upload and run code to test it. Open (or [download](https://www.arduino.cc/en/main/software) and open) the Arduino app, and open the file "photoresistor/photoresistor/ino" located in this repository. 
+```cpp
+void setup() {
+  // initialize serial communication at 9600 bits per second:
+  Serial.begin(9600);
+}
 
+void loop() {
+  int sensorValue = analogRead(A0);
+  // print out the value you read:
+  Serial.println(sensorValue);
+  delay(1);        // delay in between reads for stability
+}
 ```
-/* 
- *  Photoresistor sample code. Prints out value of photoresistor 
- *  reading on analog pin A5. 
+
+_Analog pins do not need to be configured as input pins using `pinMode(INPUT)`, as they are already automatically configured as such_
+
+### Types of sensors
+
+Analog sensors allow the Arduino to sense a voltage that indicates the sensor's current reading; for example, a thermistor is a resistor that changes resistance depending on the current temperature. By building a circuit with a thermistor, the Arduino can determine the current temperature as a function of the sensor's resistance.
+
+Digital sensors can communicate with the Arduino using a variety of protocols, such as [I2C](https://learn.sparkfun.com/tutorials/i2c) or [SPI](https://learn.sparkfun.com/tutorials/serial-peripheral-interface-spi). These protocols allow the sensor to transfer data to the Arduino. In contrast to the thermistor, which required an Arduino to read the sensor's resistance and determine the temperature, a digital temperature sensor may simply transmit the current temperature to the Arduino.
+
+### Hooking up sensors
+Wiring a sensor to an Arduino depends on the communication protocol in use. This tutorial will cover wiring with an analog sensor and a digital I2C sensor. All sensors mentioned in the tutorial are available at Treehacks. Many of them come with circuit boards that simplify the wiring.
+
+#### Analog Sensors
+We'll go over how to use one of Treehacks' photoresistors with an Arduino Uno. Take the sensor breakout and plug in one of the three pin attachment cables. This cable allows easy connection to an Arduino or breadboard. Plug the black lead (labeled GND) into a GND (ground, or 0V) pin on the Arduino, the red (labeled VCC) into the Uno's 5V pin, and the yellow/blue cable into A2.
+
+![Wiring picture](https://arduinomodules.info/wp-content/uploads/Arduino_KY-018_Keyes_photoresistor_module_connection_diagram.png)
```

Comment:
@jsutaria can you upload this picture to github instead? just in case the site goes down in the future

Reply:
Yep will do!